test/utils.py

```diff
@@ -71,7 +71,11 @@ def _copy_over_tables(tables: List[str], schema= SCHEMA, test_schema=TEST_SCHEMA
                     END $$;""")
     # Create views
     with get_db_connection(schema=test_schema) as con:
-        refresh_derived_tables_exec(con, list_views(), schema=test_schema)
+        views: List[str] = list_views()
```

Comment:
## Bug fix: `test_n3c.all_csets_view_new" does not exist`
Problem her was that I was making views in `test_n3c` incorrectly. `refresh_derived_tables_exec()` should be called to refresh derived tables, not views. Not sure why this wasn't causing an error before. It may be because `cset_members_items_plus` was the only view and it wasn't really being used.